packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -101,7 +101,9 @@ export const defaultChatItemMessageContainer = (overlapAvatarAndMessage: boolean
 export const defaultMyChatMessageContainer: ComponentSlotStyle = {
   maxWidth: '100%',
   minWidth: `${CHAT_MESSAGE_CONTAINER_MIN_WIDTH_REM}rem`,
-  marginLeft: '0rem'
+  marginLeft: '0rem',
+  // This makes message bubble show border in high contrast mode making each message distinguishable
+  border: '1px solid transparent'
```

Comment:
note: This would introduce "thicker" messages by default but it is an elegant and probably unnoticeable change by users :)

Reply:
On another note, can someone test this on a 4k/high res display? I'm curious if the 1px will cause issues